crate/get-help.md

```diff
@@ -1 +1,20 @@
-[the public Crate community on Slack](https://crate.io/docs/support/slackin/)
```

Comment:
This file is inserted in the "quick reference" section under "where to get help". It is designed for a short message, so it is going to look strange with new headers inside the [current bulleted list](https://github.com/docker-library/docs/tree/333fb8a73f7d600add68cbb3ee8c039ddd530fca/crate#quick-reference).  Sorry we haven't documented this file yet.